aspnetcore/blazor/tooling.md

```diff
@@ -690,37 +690,68 @@ The .NET WebAssembly build tools are based on [Emscripten](https://emscripten.or
 
 When [ahead-of-time (AOT) compilation](xref:blazor/host-and-deploy/webassembly#ahead-of-time-aot-compilation) is used, [WebAssembly Single Instruction, Multiple Data (SIMD)](https://github.com/WebAssembly/simd/blob/master/proposals/simd/SIMD.md) is enabled by default for all major browsers. SIMD can improve the throughput of vectorized computations by performing an operation on multiple pieces of data in parallel using a single instruction.
 
+To enable AOT, set `<RunAOTCompilation>` to `true` in the app's project file (`.csproj`):
+
 ```xml
 <PropertyGroup>
   <RunAOTCompilation>true</RunAOTCompilation>
 </PropertyGroup>
 ```
 
-:::moniker-end
+To disable SIMD, for example when targeting old or mobile devices, add the `<RunAOTCompilation>` property set to `false`:
```

Comment:
To disable SIMD, for example when targeting old browsers (on mobile devices), add the `<WasmEnableSIMD>` property set to `false`: